js_machine_front/src/styles/route.css

```diff
@@ -0,0 +1,20 @@
+.activeLink, a:active {
+    color: #F2E14C;
+}
+
+.nav {
+    position: absolute;
+    right: 3vh;
+    top: 8vh;
+}
+
+.nav__link {
+    padding-right: 8vh;
+}
+a {    

```

Comment:
Remove style for "a" tags. Better use class.

Reply:
yes